src/applications/vaos/new-appointment/components/DateTimeRequestPage.jsx

```diff
@@ -35,133 +32,150 @@ export function getOptionsByDate() {
     },
   ];
 }
-export class DateTimeRequestPage extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { submitted: false, validationError: null };
-  }
 
-  componentDidMount() {
-    document.title = `${pageTitle} | Veterans Affairs`;
-    scrollAndFocus();
-  }
+function isMaxSelectionsError(validationError) {
+  return validationError === maxSelectionsError;
+}
 
-  componentDidUpdate(prevProps, prevState) {
-    const { validationError } = this.state;
-    if (
-      validationError &&
-      (this.isMaxSelectionsError() ||
-        (!prevState.submitted && this.state.submitted))
-    ) {
-      scrollAndFocus('.usa-input-error-message');
-    }
+function userSelectedSlot(calendarData) {
+  return calendarData?.selectedDates?.length > 0;
+}
+
+function goBack({ routeToPreviousAppointmentPage, history }) {
+  return routeToPreviousAppointmentPage(history, pageKey);
```

Comment:
For a 1 liner that isn't reused, you might want to consider just calling this in line on line 153.  See comment below;